File: plain/src/main/scala/com/ibm/techempower/PlainTextResource.scala
package com.ibm.techempower

import com.ibm.plain.rest.Resource
import com.ibm.plain.text.`UTF-8`

final class PlainTextResource

  extends Resource {
  
  import PlainTextResource._

  Get { hello }

}

object PlainTextResource {
  
  final val hello = "Hello, World!".getBytes(`UTF-8`)  
  
}

File: frameworks/Java/grizzly-jersey/src/main/java/hello/PlaintextResource.java
package hello;

import static javax.ws.rs.core.MediaType.APPLICATION_JSON;

import java.util.HashMap;
import java.util.Map;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;

import com.sun.jersey.spi.resource.Singleton;

@Singleton
@Path("/plaintext")
public class PlaintextResource {

  @GET
  @Produces("text/plain; charset=utf-8")
  public Object plaintext() {
    String data = "Hello, World!";
    return data;
  }
}
